DEV-3/DEV-3/NumbersConversion.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DEV_3
+{
+    /// <summary>
+    /// Conversion a number to the selected number system
+    /// </summary>
+    class NumbersConversion
+    {
+        int decimalNumber;
+        int scaleOfNotation;
+        
+        public NumbersConversion (int arg1, int arg2)
+        {
+            decimalNumber = arg1;
+            scaleOfNotation = arg2;
+        }
+
+        /// <summary>
+        /// Conversion a number to the selected number system
+        /// </summary>
+        /// <returns> Converted number</returns>
+        public List<int> ConversionToRadix()

```

Comment:
Return int requires caller to perform additional conversion from symbols code to char